turbinia/workers/__init__.py

```diff
@@ -160,15 +161,15 @@ def save_local_file(self, file_):
       if writer.name != 'LocalOutputWriter':
         writer.write(file_)
 
-  def set_error(self, error, traceback):
+  def set_error(self, error, traceback_):
```

Comment:
Is the trailing _ intended here?

Reply:
Yeah, since I imported the traceback module, the name conflicted, so I added it.  I know it's not exactly Plaso style, but PEP8 does mention that trailing underscore is "used by convention to avoid conflicts with Python keyword".  Leading underscore seems to make linters unhappy.